src/NMM_Gateway.php

```diff
@@ -292,7 +292,7 @@ public function additional_email_details($order, $sent_to_admin, $plain_text, $e
         $orderCryptoTotal = WC()->session->get($crypto->get_id() . '_amount');
         $orderWalletAddress = get_post_meta($order->get_id(), 'wallet_address', true);
         
-        $qrCode = $this->get_qr_code($crypto, $orderWalletAddress, $orderCryptoTotal);
+        $qrCode = $this->get_qr_code($crypto, $orderWalletAddress, $orderCryptoTotal, $orderId);
```

Comment:
$orderId isn't available here, that should be $order->get_id()

    $qrCode = $this->get_qr_code($crypto, $orderWalletAddress, $orderCryptoTotal, $order->get_id());